network.go

```diff
@@ -1171,8 +1183,8 @@ func (n *network) createEndpoint(name string, options ...EndpointOption) (Endpoi
 	if err = ep.assignAddress(ipam, false, n.enableIPv6 && n.postIPv6); err != nil {
 		return nil, err
 	}
-
-	if err = n.getController().updateToStore(ep); err != nil {
+	//Commented out and calling same API before calling addEndPoint
+	/*if err = n.getController().updateToStore(ep); err != nil {
```

Comment:
How about removing this code block instead of keeping it commented? Since git will store the previous version of the code, good comments alone may be sufficient. Commented code blocks are somewhat rendundant (and potentially confusing).

Reply:
ok. Will wait for other's comment and will incorporate all review comments together.

Reply:
yep +1 for that